pkg/e2e/e2e.go

```diff
@@ -650,6 +644,13 @@ func runTestsInPhase(phase string, description string, dryrun bool) bool {
 	phaseReporter := ginkgorep.NewPhaseReporter(phase, phaseReportPath)
 	ginkgoPassed := false
 
+	if !dryrun || ginkgoConfig.GinkgoConfig.DryRun {
```

Comment:
So, we only want this to happen if it's *not* a dryrun, right? Shouldn't it be:

```suggestion
	if !dryrun || !ginkgoConfig.GinkgoConfig.DryRun {
```